Description: remove broken hardcoded install-layout from setup.py
Author: Dmitry Shachnev <mitya57@ubuntu.com>
Forwarded: no
Last-Update: 2014-06-16

--- a/setup.py
+++ b/setup.py
@@ -9,10 +9,6 @@
 import meld.conf
 
 
-if (platform.system() == 'Linux' and
-        platform.linux_distribution()[0] == 'Ubuntu'):
-    sys.argv.append('--install-layout=deb')
-
 setup(
     name=meld.conf.__package__,
     version=meld.conf.__version__,
